test/sql/aggregate/group/tpch_group_by_all.test_slow

```diff
@@ -0,0 +1,117 @@
+# name: test/sql/aggregate/group/tpch_group_by_all.test_slow
+# description: Test TPC-H SF0.01 GROUP BY ALL
+# group: [group]
+
+require tpch
+
+statement ok
+CALL dbgen(sf=0.01);
+
+query I
+SELECT
+    l_returnflag,
+    l_linestatus,
+    sum(l_quantity) AS sum_qty,
+    sum(l_extendedprice) AS sum_base_price,
+    sum(l_extendedprice * (1 - l_discount)) AS sum_disc_price,
+    sum(l_extendedprice * (1 - l_discount) * (1 + l_tax)) AS sum_charge,
+    avg(l_quantity) AS avg_qty,
+    avg(l_extendedprice) AS avg_price,
+    avg(l_discount) AS avg_disc,
+    count(*) AS count_order
+FROM
+    lineitem
+WHERE
+    l_shipdate <= CAST('1998-09-02' AS date)
+GROUP BY ALL
+ORDER BY
```

Comment:
Maybe `ORDER BY *` here to stress that code path?

Reply:
Sure, sounds good!